server/grpc.go

```diff
@@ -22,15 +21,15 @@ type StreamServer struct {
 	errorLogger   *zap.Logger
 }
 
-func NewStreamServer(pubsub pubsub.PubSuber, accessLogger *zap.Logger, errorLogger *zap.Logger, config config.Config) *StreamServer {
+func NewStreamServer(opt Option) *StreamServer {
```

Comment:
__[golint]__
* [golint] exported function NewStreamServer should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->